lib/atsec_test.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Base module for audit-test test cases
```

Comment:
It might be ATSEC/atsec not audit, right?

Reply:
fixed